README.md

```diff
@@ -34,8 +34,9 @@ group.background {
 group.background {
     print("This is also run on the background queue in parallel")
 }
-group.wait()
-print("Both asynchronous blocks are complete")
+group.notify {
+	print("Both asynchronous blocks are complete")
+}
```

Comment:
Thanks for your reply. What about this?
~~~swift
//The notify block run on the main queue by default, you can specify the custom queue.
group.notify {
	print("Both asynchronous blocks are complete")
}
~~~